cms/templates/certificates.html

```diff
@@ -106,6 +106,9 @@ <h3 class="title-3">${_("Issuing Certificates to Learners")}</h3>
             grading_url = utils.reverse_course_url('grading_handler', context_course.id)
             course_team_url = utils.reverse_course_url('course_team_handler', context_course.id)
             advanced_settings_url = utils.reverse_course_url('advanced_settings_handler', context_course.id)
+            exams_url = ''
+            if settings.FEATURES.get('EXAM_SETTINGS_HTML_VIEW'):
```

Comment:
This looks like it's acting as the feature toggle on the link, based on the way it's used, but the name doesn't suggest that to me. I think something like `ENABLE_EXAM_SETTINGS_HTML_VIEW` would be more clear. What do you think?

Reply:
I think that makes sense